lms/djangoapps/django_comment_client/management/tests.py

```diff
@@ -0,0 +1,248 @@
+"""
+UnitTests for the rename_user function in the management_utils module
```

Comment:
Shouldn't this file live in `lms/djangoapps/django_comment_client/tests/test_management_utils.py`?